documentation/modules/installing-openshift.adoc

```diff
@@ -0,0 +1,80 @@
+[[installing-openshift]]
+
+== Installing {ProductName} on OpenShift
+
+[[prerequisites]]
+=== Prerequisites
+
+To install {ProductName}, you need to have the OpenShift client tools. You can download OpenShift
+Origin client from https://github.com/openshift/origin/releases[OpenShift Origin]. {ProductName} is
+tested to work with the latest stable release.
+
+If you do not have an OpenShift cluster available, see
+https://github.com/minishift/minishift[minishift] for an example of how to run a local instance of OpenShift
+on your machine.
+
+.Procedure 
+
+. Download one of the releases from https://github.com/EnMasseProject/enmasse/releases and unpack it.
+
+Once unpacked, you can deploy {ProductName} using the deployment script provided in the release.
+
+You can invoke the deployment script with `-h` to get a list of options.
+
+{ProductName} can be deployed in either a single-tenant mode or a multi-tenant mode. Choose
+single-tenant mode if you do not expect to be creating multiple isolated address spaces or if you do
+not have cluster-admin privileges. Choose multi-tenant mode otherwise.
+
+==== Installing in single-tenant mode
+
+.Procedure 
+
+. To install in single-tenant mode run the following script:
+
++
+[source,options="nowrap"]
+----
+./deploy-openshift.sh -m "https://localhost:8443" -n enmasse
+----
++
+Running this script creates the deployments required for running {ProductName} in single-tenant mode. Starting
+up {ProductName} will take a while, usually depending on how fast it is able to download the Docker
+images for the various components.
+
+. You can use the REST API to check the status of the deployment:
++
+[source,options="nowrap"]
+----
+curl http://$(oc get route restapi -o jsonpath='{.spec.host}')/v1/addressspaces/default
+----
++
+You can consider the deployment to be finished when `status.isReady` is `true` in the returned JSON
+object.
+
+==== Installing in multi-tenant mode
```

Comment:
Same as above; add "{ProductName}": "Installing {ProductName} in Multi-Tenant Mode"

Also, add the paragraph from above about whether to choose single-tenant or multi-tenant immediately under this header